statsmodels/tsa/stattools.py

```diff
@@ -628,13 +629,17 @@ def acf(
     Returns
     -------
     acf : ndarray
-        The autocorrelation function.
+        The autocorrelation function, nlags+1 elements including lag zero.
     confint : ndarray, optional
-        Confidence intervals for the ACF. Returned if alpha is not None.
+        Confidence intervals for the ACF, 2*(nlags+1) elements including
```

Comment:
Can you change here `Confidence intervals for the ACF. Has shape (nlags+1, 2) since it includes`

Reply:
Done, thanks